README.rst

```diff
@@ -10,6 +10,10 @@ The Reactive Extensions for Python (RxPY)
 
 .. image:: https://img.shields.io/pypi/v/rx.svg
         :target: https://pypi.python.org/pypi/Rx
+        
+.. image:: https://readthedocs.org/projects/rxpy/badge/?version=latest
```

Comment:
To be consistent with the other badges, this could also use the img.shields.io service?

https://img.shields.io/readthedocs/:packageName.svg

Reply:
Or, including the version,
https://img.shields.io/readthedocs/:packageName/:version.svg

Reply:
Thanks, should be fixed now

Reply:
would it be better to point at the later docs (from the readme for master, after all)?

```
.. image:: https://img.shields.io/readthedocs/rxpy/latest.svg
        :target: https://rxpy.readthedocs.io/en/latest/
        :alt: Documentation Status
```

Doesn’t matter much, I guess.